### PR TITLE
Fixes Spelling

### PR DESCRIPTION
**Pull request by jimmycasey**

### CODE REVIEW

**Comment by scw00** [comment]
[approve ci]

File: LICENSE [hunk 1]
```diff
@@ -545,7 +545,7 @@ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
  * SUCH DAMAGE.
  *
- * The licence and distribution terms for any publically available version or
```
Comment (masaori335) [inline LICENSE]: This is text of "Original SSLeay License". We should keep this text even if it's typo.
https://www.openssl.org/source/license.html

Comment (jimmycasey) [reply LICENSE]: will do

**Comment by SolidWallOfCode** [comment]
[approve ci]

**Pull request merged**